packages/lib/src/project/AppProject.js

```diff
@@ -1,8 +1,9 @@
-import Project from "./Project";
+import BasePackageProject from "./BasePackageProject";
 import VersionedApp from "../app/VersionedApp";
 import Package from "../package/Package";
+import _ from 'lodash';
 
-export default class AppProject extends Project {
+export default class AppProject extends BasePackageProject {
```

Comment:
I think we can use just `PackageProject`

Reply:
Hmm I actually prefer to make it clear which are base classes, and which are leaves that can be actually instantiated.